examples/application.rs

```diff
@@ -696,10 +660,16 @@ impl WindowState {
         // Allow IME out of the box.
         let request_data = ImeRequestData::default()
             .with_purpose(ImePurpose::Normal)
+            // The input field is not displayed at all in this demo
+            .with_hint(ImeHint::HIDDEN_TEXT)
             .with_cursor_area(LogicalPosition { x: 0, y: 0 }.into(), IME_CURSOR_SIZE.into())
```

Comment:
I think maybe rip that off as well, just to make it more easier to follow and given that we have IME example dedicated to it. I'd also probably move comment wrt enabling IME here to refer to `ime` example, WDYT?

Reply:
Oh, sorry, I missed that.
I'm not sure about having the comment about missing functionality in the middle of the code. Yes, that's the initialization code, but initialization is not always where people look for APIs - sometimes APIs are enabled by hooks, for example. That's why I put the comment at the top. I'm also not sure about placing it there, but I hope that's where people looking for general guidance look.

Reply:
Removed as requested.